keyboards/scottokeebs/scotto34/info.json

```diff
@@ -0,0 +1,73 @@
+{
+    "manufacturer": "ScottoKeebs",
+    "keyboard_name": "Scotto34 (PCB Edition)",
+    "maintainer": "joe-scotto",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["GP9", "GP10", "GP11", "GP12", "GP13", "GP18", "GP19", "GP20", "GP21", "GP22"],
+        "rows": ["GP0", "GP1", "GP14", "GP15"]
+    },
+    "processor": "RP2040",
+    "url": "https://scottokeebs.com",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x3334",
+        "vid": "0x534B"
+    },
+    "ws2812": {
+        "pin": "GP23",
+        "driver": "vendor"
+    },
+    "rgblight": {
+        "led_count": 1
+    },
```

Comment:
one more thing 🙊
`"community_layouts": ["split_3x5_2"],`

seeing as this keyboard is coming new to QMK 😉

Reply:
Added.